geode-assembly/build.gradle

```diff
@@ -384,7 +384,7 @@ tasks.register('gfshDepsJar', Jar) {
 
 tasks.register('docs', Javadoc) {
   def docsDir = file("$buildDir/javadocs")
-  options.addStringOption('Xdoclint:none', '-quiet')
+  options.addStringOption('Xwerror', '-quiet')
```

Comment:
Does it break, cause failures for other devs/PRs?

Reply:
No, the build currently passes, and will only fail if someone breaks existing javadocs or introduces new javadocs that have errors or warnings.